lib/graphql/account.ts

```diff
@@ -13,10 +13,31 @@ const GET_ACCOUNT = `query {
     }
   }
 }`;
+const GET_ACCOUNT_STATS = `query {
+  viewer {
+    mainAccount {
+      accountStats {
+        accountBalance
+				main
```

Comment:
In VS code this doesnt look that weird 🤔

Reply:
these are tabs instead of spaces 🙂